tsconfig.json

```diff
@@ -4,7 +4,6 @@
     "noUnusedLocals": true,
     "experimentalDecorators": true,
     "skipLibCheck": true,
-    "strictNullChecks": true,
```

Comment:
We would like to have strict null checks if possible.

What triggered you to remove it?

Reply:
Bringing over some of the validators from cmc-citizen-frontend would not compile with strictNullChecks on. I shall have a look into it and get them working with it on.

Reply:
Looking into this, it was possible to rework the validators to work with strictNullChecks so this has been put back in.

Reply:
Nice. We would like to enable strict null checks in frontends at some point. This will be great first step.